gridData/tests/test_dx.py

```diff
@@ -66,6 +76,51 @@ def test_write_dx(tmpdir, nptype, dxtype, counts=100, ndim=3):
 
     assert_equal(out_dxtype, dxtype)
 
+@pytest.mark.parametrize("nptype,dxtype", [
```

Comment:
Avoid code duplication, just parametrize the other test for the uncompressed file over the compressed and uncompressed files.